test/Microsoft.AspNetCore.SignalR.Redis.Tests/RedisHubLifetimeManagerTests.cs

```diff
@@ -522,13 +524,14 @@ public class RedisHubLifetimeManagerTests
             {
                 // Force an exception when writing to connection
                 var output = new Mock<Channel<HubMessage>>();
-                output.Setup(o => o.Out.WaitToWriteAsync(It.IsAny<CancellationToken>())).Throws(new Exception());
+                output.Setup(o => o.Out.WaitToWriteAsync(It.IsAny<CancellationToken>())).Throws(new Exception("Message"));
 
                 var connection = new HubConnectionContext(output.Object, client.Connection);
 
                 await manager.OnConnectedAsync(connection).OrTimeout();
 
-                await Assert.ThrowsAsync<Exception>(() => manager.InvokeConnectionAsync(connection.ConnectionId, "Hello", new object[] { "World" }).OrTimeout());
+                var exception = await Assert.ThrowsAsync<Exception>(() => manager.InvokeConnectionAsync(connection.ConnectionId, "Hello", new object[] { "World" }).OrTimeout());
+                Assert.Equal("Message", exception.Message);
```

Comment:
I was thinking more about just storing the exception in a variable and then using `Assert.Same()` but this is fine too.